README.md

```diff
@@ -183,7 +183,7 @@ Inform the controller about the Thread credentials it needs to use when commissi
 ```
 
 **Commission with code**
-Commission a new device. For WiFi or Thread based devices, the credentials need to be set upfront, otherwise commisisoning will fail.
+Commission a new device. For WiFi or Thread based devices, the credentials need to be set upfront, otherwise commisisoning will fail. Supports both QR-code syntax (MT:...) and manual pairing code as string.
```

Comment:
```suggestion
Commission a new device. For WiFi or Thread based devices, the credentials need to be set upfront, otherwise commissioning will fail. Supports both QR-code syntax (MT:...) and manual pairing code as string.
```